packages/iobio-react-components/package.json

```diff
@@ -8,14 +8,16 @@
 	"scripts": {
 		"dev": "vite",
 		"build": "tsc -b && vite build",
+		"indexer": "pnpm exec tsx --env-file=.env src/utils/elasticMetadataClient/index.mts",
 		"lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
 		"preview": "vite preview",
-		"stats": "pnpm exec tsx --env-file=.env src/utils/stats.mts"
+		"stats": "pnpm exec tsx --env-file=.env src/utils/statisticsClient/index.mts"
```

Comment:
Open to naming improvements.

The `statisticsClient` utility generates the Iobio metadata.

The `indexer` utility updates the ElasticSearch index & document (and hooks into the stats util, so it can all be done together)

Reply:
In addition it's worth mentioning that these are Node utilities in the `iobio-react-components` package, so maybe a follow up to reorg is in order.

Reply:
@demariadaniel good suggestion on reorg I think...
the whole setup here in "react-components" is a Vite project while this work is a Node client with no build step

even just the weirdness around not needing file extensions for say components https://github.com/overture-stack/iobio-components/blob/main/packages/iobio-react-components/src/components/histogram.tsx (because we have a build tool) vs needing them in the stats client https://github.com/overture-stack/iobio-components/pull/77#discussion_r2237066651